### PR TITLE
docs: bump to v4.0.0 + minor doc adjustments

### LINKED ISSUE #2: Update various documentation & README
Not sure how to convert the README.erb.md to a normal README.md

Let's leverage all that GH has to offer for these things.. wiki, pages, etc...

### PR DESCRIPTION
**Pull request by gegles**
Partially resolves #2  (which was too vague anyways)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,26 @@
...
 
 Disclaimers:
 
-* Aspera, FASP are owned by IBM
-* This GEM is not endorsed/supported by IBM/Aspera
-* Use at your risk (not in production environments)
-* This gem is provided as-is, and is not intended to be a complete CLI, or industry-grade product.
-* some features may not be fully validated
-* IBM provides an officially supported Aspera CLI: [http://downloads.asperasoft.com/en/downloads/62](http://downloads.asperasoft.com/en/downloads/62) .
+* This as not yet been officially released so things may change
```
Comment (dwosk) [inline README.md:19]: minor typo here

Comment (gegles) [inline README.md:19]:
```suggestion
* This has not yet been officially released so things may change
```

File: docs/README.erb.md [hunk 2]
```diff
@@ -11,23 +11,19 @@ _Laurent/2016-<%=Time.new.year%>_
...
 
 Disclaimers:
 
-* Aspera, FASP are owned by IBM
-* This GEM is not endorsed/supported by IBM/Aspera
-* Use at your risk (not in production environments)
-* This gem is provided as-is, and is not intended to be a complete CLI, or industry-grade product.
-* some features may not be fully validated
-* IBM provides an officially supported Aspera CLI: [http://downloads.asperasoft.com/en/downloads/62](http://downloads.asperasoft.com/en/downloads/62) .
+* This as not yet been officially released so things may change
```
Comment (dwosk) [inline docs/README.erb.md:19]: same typo here

File: README.md (hunk 1 above)
Comment (gegles) [reply README.md]: > minor typo here

Thanks! GH is cool, with a suggestion (from me or you), I just had to click on accept suggestion and bam! it's committed and the PR updated...

**Review by dwosk — APPROVED** [review]

File: docs/README.erb.md (hunk 2 above)
Comment (gegles) [reply docs/README.erb.md:19]:
```suggestion
* This has not yet been officially released so things may change
```
Ah ah, silly me, I forgot to change the actual source of the issue...

**Pull request merged**